qa/t/webservice_bug_get.t

```diff
@@ -16,7 +16,7 @@ use Data::Dumper;
 use DateTime;
 use QA::Util;
 use QA::Tests qw(bug_tests PRIVATE_BUG_USER);
-use Test::More tests => 1006;
+use Test::More tests => 1087;
```

Comment:
That is the number of tests that it expects to run. When you add code to do more or less checks, you have to adjust that number. If more or less tests are run than what is expected than it will throw a failure for the script. Newer scripts that I write do not worry about the number and omit this part making it more dynamic.